app/web/components/ContributorForm/ContributorForm.test.tsx

```diff
@@ -40,36 +40,46 @@ describe("contributor form", () => {
   });
 
   it("can be submitted filled", async () => {
-    jest.setTimeout(10000);
     const processForm = jest.fn(() => Promise.resolve());
     render(<ContributorForm processForm={processForm} />, { wrapper });
-    userEvent.type(screen.getByLabelText(IDEAS_LABEL), "I have great ideas");
-    userEvent.type(
+
+    const user = userEvent.setup();
+
+    await user.type(screen.getByLabelText(IDEAS_LABEL), "I have great ideas");
+    await user.type(
       screen.getByLabelText(FEATURES_LABEL),
       "I want all the features"
     );
-    userEvent.click(screen.getByRole("radio", { name: "Yes" }));
-    // For some reason, checking boxes makes tests flakey when running jest
```

Comment:
Has the upgrade somehow magically fixed the performance issue?

Reply:
Yes! But also I think probably using `waitFor` back then might have fixed it too. Since these return promises now it's easier to know it's done clicking before you check the assertion.